tests/framework/installer/ceph_manifests.go

```diff
@@ -349,9 +357,10 @@ parameters:
   csi.storage.k8s.io/node-stage-secret-namespace: ` + m.settings.Namespace + `
 `
 	if m.settings.ConnectionsEncrypted {
-		// encryption requires either the 5.11 kernel or the fuse mounter. Until the newer
-		// kernel is available in minikube, we need to test with fuse.
-		sc += "  mounter: fuse"
+		// encryption with kernel version <= 5.11 requires `mounter: fuse` or with kernel version >= 5.12
+		// it requires `mounter: kernel`. Since, github minikube has kernel version 5.12 setting it to
+		// mounter with kernel.
```

Comment:
```suggestion
// Encryption with kernel version <= 5.11 requires 'mounter: fuse'. For kernel version >= 5.12, it requires 'mounter: // kernel'. Since the Github Minikube has kernel version > 5.12, the setting is set to 'mounter: kernel'.
```